blog/2022/12-7-first-look-at-resoto-dashboards/index.md

```diff
@@ -0,0 +1,76 @@
+
+This dashboard has pre-configured a lot of widgets with relevant information about your cloud infrastructure.
+
+![ui-dashboard-manager](./img/ui-example-dashboard.png)
+
+Let's take a quick look at some of the elements being displayed here. Look for the table widget titled "Unused Volumes older than 1 week". There you have a complete list, as the title suggests, of the available volumes older than 1 week. This information is really useful if you want to know what resources you may mark for cleanup, and is very easily retrieved with resoto and displayed in a dashboard.
+
+To see how you can make this information available in your dashboards, let's check that specific widget's configuration. Pressing the edit button in the top-right corner of the dashboard, you can toggle the _edit mode_. While in this mode, you can move and resize your widgets, edit their settings to modify what they are showing and how they are displaying the information. You will note that an edit button appeared also on each widget. Press the one in the table we are inspecting. The following dialog should pop up:
+
+![ui-edit-table](./img/ui-edit-table.png)
+
+In the right part of this dialog, you can see a preview of the widget with the data and, below it, some display options (just some color settings in this case). In the left part, we have a more important section. There you can define the _data sources_ to retrieve the data you want to show in the widget. In this case, a quite simple _full-text search_ command was used. You can learn more about _full-text search_ in resoto [here](https://resoto.com/docs/reference/search/full-text).
+
+You can modify that search to fit your needs. As a little exercise, try to make the query show only the volumes in the "eu-central-1" region which are older than 10 days. Once you are finished with your modifications, press "Accept" and the widget will be updated in the dashboard.
+
+Finally, toggle the edit mode on the top-right corner of the dashboard to return to the normal mode.
+
+### Checking out volumes size
+
+Another interesting widget to check out is the chart that is titled "Volumes Sizes". This widget shows how the size of the volumes grew over time for each account. Hovering the mouse over it will show a legend with summarized information about the data being displayed (in this case, volume size per account). Let's enter into the edit mode again, and then press the edit button in this widget, to check how we made it display this information.
+
+![ui-edit-chart](./img/ui-edit-chart.png)
+
+This time we are using a _time series data source_. This data source gets the data from **prometheus** time-series database. This information is available there because we previously configured some metrics in the `resoto.metrics` configuration. For more information about this, check [this blog post](https://resoto.com/blog/2022/06/09/building-actionable-cloud-infrastructure-metrics#how-metrics-are-made).
```

Comment:
Maybe mention that it gets the data from "the Prometheus time-series database that ships as part of Resoto".